### DISCUSSION TITLE
Is there a typescript version of react-aria primitives?

### DISCUSSION THREAD

**Discussion by nimeshvaghasiya** [adobe/react-spectrum#4107] [General]
as it makes way harder to implement it in typescript.

**Comment by reidbarber** [comment]
React Aria is written in typescript, and any types you would need to use are exported. Examples in the docs, however, currently aren't written in typescript, but work to add that is in progress at https://github.com/adobe/react-spectrum/pull/3710. You can also check the react-spectrum implementations, which are written in typescript.
